packages/feedback/src/widget/createWidget.ts

```diff
@@ -94,6 +94,18 @@ export function createWidget({
       return;
     }
 
+    // Simple validation for now, just check for non-empty message if name required
+    if (options.isNameRequired && !feedback.name) {
+      dialog.showError('Please enter in a name before submitting!');
+      return;
+    }
+
+    // Simple validation for now, just check for non-empty message if email required
+    if (options.isEmailRequired && !feedback.email) {
+      dialog.showError('Please enter in an email before submitting!');
+      return;
+    }
+
```

Comment:
We may want to check all fields at once instead of sequentially, otherwise user may end up having to fill out each field individually if they were to follow only the error message.

We could display a message like... `Please enter in the following required fields: name, description` (we may want to grab the field names from options object using the labels).